token/js/package.json

```diff
@@ -49,7 +49,7 @@
         "deploy:docs": "npm run docs && gh-pages --dist token/js --dotfiles"
     },
     "peerDependencies": {
-        "@solana/web3.js": "^1.20.0"
+        "@solana/web3.js": "^1.47.4"
```

Comment:
Is a reason these minimum versions changed? It doesn't matter hugely, but in general it seems ideal to declare a lower minimum version if it's not needed (1.20 is kind of a magic number I saw in the wild a bunch). Just wondering if this relies on some specific API that became available as of this version.

Reply:
so i was upgrading it because of ```returnData``` in ```SimulatedTransactionResponse``` which was added just a month ago , it was not showing before in lower version of ```web3.js``` but idk why when installing lower version now i checked it is having ```returnData``` so i think we can just let it be at ```1.20.0```. Maybe by mistake i saw something else at that time.

Reply:
Ah, no that's a good point. If we need that API let's require a newish version that has it!

Reply:
i am actually confused why i am getting ```returnData``` in ```1.20.0```  even though here is the PR that merged for it https://github.com/solana-labs/solana/pull/26499 , do all merge appears in lower version also 🤔 or is there some problem with my build ?

Reply:
So if your version is set with ^ (as it is in this PR's dev dependencies) that will still resolve to the most recent matching version of 1.x.x.

You can see what version is actually being resolved to in the package-lock.json file.

Reply:
oh yes, correct. Thanks for this 🙂 , spend a lot of time in figuring this out.

Reply:
I think this is good to merge as is then?

Reply:
yes :)